puppet/modules/prprocessor/manifests/init.pp

```diff
@@ -65,6 +64,15 @@
     require     => Package[$packages],
   }
 
+  cron { 'close inactive':
+    command     => "cd ${app_root} && bundle exec ruby scripts/close_inactive.rb",
+    user        => $username,
+    environment => ["HOME=${app_root}"],
+    hour        => 1,
+    minute      => 23,
+    require     => Exec['install prprocessor']
```

Comment:
dont you need the ENV=GITHUB_OAUTH_TOKEN too?

Reply:
I'm not sure, where would i get the token from?

Reply:
You could do `GITHUB_OAUTH_TOKEN='${github_oauth_token}' bundle exec`. I think cron files per user can be considered secure since you can't read other users' crontabs.